src/main/java/org/mozilla/msrp/platform/vertical/content/ContentController.kt

```diff
@@ -45,11 +45,11 @@ class ContentController @Inject constructor(private val contentService: ContentS
             })
 
 
-    @RequestMapping("/api/v1/content")
+    @GetMapping("/api/v1/content")
     fun getContent(
             @RequestParam(value = "category") category: String,
             @RequestParam(value = "locale") locale: String,
-            @RequestParam(required = false) tag: String = ""
+            @RequestParam(required = false) tag: String? = ""
```

Comment:
Could use `defaultValue = ""` to get rid of null-able tag string
https://github.com/mozilla-tw/mango_backend/blob/develop/src/main/java/org/mozilla/msrp/platform/vertical/video/VideoController.kt#L31

Reply:
hmm..I want it to be nullable. So the default value can be removed.